2024.03/alpine/Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM alpine:3.19
+FROM alpine:latest
 
 RUN addgroup -S raku && adduser -S raku -G raku
 
-ARG rakudo_version=2024.02-01
-ENV rakudo_version=${rakudo_version}
+ARG rakudo_version=2024.03
```

Comment:
Just for clarity... my intention to split the former "rakudo_version", was a future automation, I have in mind.
In it, I would check existing "git tags" and / or available "latest" Rakudo Star releases, which do **NOT** have the "build_revision"... and based on those checks, I'd update (say RegEx) the Docker files...

Some years ago, I think I saw a "rakudo.org" API, which also returned a (or all?) "Star" release(s). If someone points to to that one __AND__ this one returns a "latest" Star version info with the "revisions", I could maybe also use this API in a comming automation.. and then leave the string as it is / was?!?

Reply:
well, just remembered... I do already something similar at https://github.com/rakudo/star/blob/a4653f4c34fad5b83b4ce3b27fa2a2ff27f8c57d/.github/workflows/publish_rakudo-star_on_rakudo.org.yml#L29

So, if you really are against splitting the "rakudo_version" and want to keep it as is, with the revision info, I know a way how to do that ;)

Reply:
Not really against doing that *per se*, but I'd prefer an alternative that would not involve changing other lines elsewhere.